[2021-09-01 03:15:47,229] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.processing_tasks.training_model_a 2021-09-01T03:15:40.142178+00:00 [queued]>
[2021-09-01 03:15:47,386] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.processing_tasks.training_model_a 2021-09-01T03:15:40.142178+00:00 [queued]>
[2021-09-01 03:15:47,387] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-09-01 03:15:47,388] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-09-01 03:15:47,396] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-09-01 03:15:47,460] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): processing_tasks.training_model_a> on 2021-09-01T03:15:40.142178+00:00
[2021-09-01 03:15:47,470] {standard_task_runner.py:52} INFO - Started process 115299 to run task
[2021-09-01 03:15:47,510] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_dag', 'processing_tasks.training_model_a', '2021-09-01T03:15:40.142178+00:00', '--job-id', '32', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmpd0ysd5q1', '--error-file', '/tmp/tmpiv61vpd5']
[2021-09-01 03:15:47,516] {standard_task_runner.py:77} INFO - Job 32: Subtask processing_tasks.training_model_a
[2021-09-01 03:15:47,885] {logging_mixin.py:104} INFO - Running <TaskInstance: xcom_dag.processing_tasks.training_model_a 2021-09-01T03:15:40.142178+00:00 [running]> on host airflowvm
[2021-09-01 03:15:48,424] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_dag
AIRFLOW_CTX_TASK_ID=processing_tasks.training_model_a
AIRFLOW_CTX_EXECUTION_DATE=2021-09-01T03:15:40.142178+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-09-01T03:15:40.142178+00:00
[2021-09-01 03:15:48,436] {logging_mixin.py:104} INFO - model's accuracy: 6.763747119039037
[2021-09-01 03:15:48,549] {python.py:151} INFO - Done. Returned value was: None
[2021-09-01 03:15:48,614] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=processing_tasks.training_model_a, execution_date=20210901T031540, start_date=20210901T031547, end_date=20210901T031548
[2021-09-01 03:15:48,814] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-09-01 03:15:48,855] {local_task_job.py:151} INFO - Task exited with return code 0
